s3watcher/lib/Transfer.js

```diff
@@ -0,0 +1,51 @@
+const S3Helper     = require('./S3Helper');
+const Upload       = require('./Upload');
+
+
+module.exports = {
+    init: function(s3Event, config){
+        const operation = function() {
+            console.log("Downloading from ingest bucket.");
+
+            const upload = Upload.buildUpload(config, s3Event);
+
+            return S3Helper.getS3Object({
+                Bucket: s3Event.bucket,
+                Key: s3Event.key
+            }).flatMap(function(data){
+                console.log("Uploading to image-loader.");
+
```

Comment:
Could put in a TODO hint to try and use the stream API for better performance (with the caveat that it's a big PITA)?